java/core/src/main/java/sleeper/core/util/ExponentialBackoffWithJitter.java

```diff
@@ -65,7 +65,7 @@ public long waitBeforeAttempt(int attempt) throws InterruptedException {
     private long getWaitMillisBeforeAttempt(int attempt) {
         double waitCeilingInSeconds = Math.min(
                 waitRange.maxWaitCeilingSecs,
-                waitRange.firstWaitCeilingSecs * 0.5 * Math.pow(2.0, attempt));
+                waitRange.firstWaitCeilingSecs * 0.5 * Math.pow(2.0, attempt - 1));
```

Comment:
Could be simplified to `Math.pow(2.0, attempt - 2)` ?

Reply:
Done, thanks